config/initializers/assets.rb

```diff
@@ -2,7 +2,7 @@
 
 # Version of your assets, change this if you want to expire all your assets.
 Rails.application.config.assets.version = '1.0'
-
+Rails.application.config.assets.precompile += ['tournaments/*']
```

Comment:
Merge master into your branch - I just merged this change in. Let's make sure we don't duplicate it.